Gruntfile.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * flopmang
+ * https://github.com/dylancwood/flopmang
+ *
+ * Copyright (c) 2014 Dylan Wood
+ * Licensed under the MIT license.
+ */
+
+'use strict';
+
+module.exports = function (grunt) {
+
+    // Project configuration.
+    grunt.initConfig({
+        watch: {
+            dev: {
+                files: ['lib/*.js', 'test/*.js'],
+                tasks: ['jshint', 'test']
+            }
+        },
+
+        jshint: {
+            all: [
+                'Gruntfile.js',
+                'lib/*.js',
+                'test/**.js'
+            ],
+            options: {
+                jshintrc: '.jshintrc',
+            },
+        },
+
+        jscs: {
+            src: {
+                options: {
+                    config: '.jscs.json'
+                },
+                files: {
+                    src: [
+                        'Gruntfile.js',
+                        'lib/*.js',
+                        'test/**.js'
+                    ],
+                }
+            },
+        },
+
+        mochacli: {
+            options: {
+                files: 'test/*_test.js'
+            },
+            spec: {
+                options: {
+                    reporter: 'spec',
+                    timeout: 10000
+                }
+            }
+        },
+
+        bump: {
```

Comment:
Hey Chris, here's what I understand of how to use bump. You would update the appropriate files, which includes a changelog, then grunt bump to auto-increment the version in package.json, tag the repo, and push to remote. Bump just makes the last step a bit easier, you can also tell bump to do a couple diff things including using major/minor/patch release numbers. I see using Bump as a way to be more consistent about semver/making life easier